src/controller/DevicePairingDelegate.h

```diff
@@ -114,6 +114,29 @@ class DLL_EXPORT DevicePairingDelegate
      * in order to resume the commissioning process.
      */
     virtual void OnScanNetworksFailure(CHIP_ERROR error) {}
+
+    /**
+     * @brief
+     *  Called when the ICD symmetric key is required.
```

Comment:
But... it expects more things to be provided than just the key, no?  The documentation needs to match reality: either we document that you have to provide all three of key, subject, check-in node, or document what you must vs can provide and default the non-mandatory things as needed.